test/integration/test_containerized_jobs.py

```diff
@@ -10,38 +10,56 @@
 
 from galaxy.util.commands import which
 from galaxy_test.base.populators import DatasetPopulator
-from galaxy_test.driver import integration_util
+from galaxy_test.driver.integration_util import IntegrationTestCase
 from .test_job_environments import BaseJobEnvironmentIntegrationTestCase
 
 SCRIPT_DIRECTORY = os.path.abspath(os.path.dirname(__file__))
+
+# local_docker
+# local_docker_inline_container_resolvers (using only fallback resolver)_
 DOCKERIZED_JOB_CONFIG_FILE = os.path.join(SCRIPT_DIRECTORY, "dockerized_job_conf.yml")
+# define an environment (local_singularity) for local execution with singularity enabled
 SINGULARITY_JOB_CONFIG_FILE = os.path.join(SCRIPT_DIRECTORY, "singularity_job_conf.yml")
+
 EXTENDED_TIMEOUT = 120
```

Comment:
This is much too long for the number of tests we're doing here.

Reply:
A rough comparison with the the integration test run for the merge commit (https://github.com/galaxyproject/galaxy/actions/runs/5327223927/) and a recent run on dev before these changes (https://github.com/galaxyproject/galaxy/actions/runs/5294112030/) seems to suggest that's 20 to 30 minutes

Reply:
Should be mostly due to the tests for the `*build*` resolvers.